lib/memo_wise.rb

```diff
@@ -3,6 +3,46 @@
 require "memo_wise/version"
 
 module MemoWise
-  class Error < StandardError; end
-  # Your code goes here...
+  def initialize(*values)
```

Comment:
Can we have YARD doc of the `initialize` method, including an `@example` section of how it works after you `prepend` to a class and then call `.new`?

Reply:
Deferring to a future PR!